tests/test_func.py

```diff
@@ -158,11 +162,12 @@ def __test_functionality(module_file, messages_file, dependencies):
 
 
 if __name__ == "__main__":
-    if "-u" in sys.argv:
-        UPDATE = True
-        sys.argv.remove("-u")
-
+    if UPDATE_OPTION in sys.argv:
+        UPDATE_FILE.touch()
+        sys.argv.remove(UPDATE_OPTION)
     if len(sys.argv) > 1:
         FILTER_RGX = sys.argv[1]
         del sys.argv[1]
     pytest.main(sys.argv)
+    if UPDATE_FILE.exists():
```

Comment:
Is the intention here to delete it only after the tests complete successfully or in all the cases, including test failures? Worth putting in a `try-finally` if it's the latter.

Reply:
Fixed in https://github.com/PyCQA/pylint/pull/4004/commits/98dc5caeb3c737b9c1340cfe212de63986ac908c